main.js

```diff
@@ -2,3 +2,57 @@
 // Go back to the lesson if you feel lost
 // Read the docs well, they usually have pretty well documented examples
 // Consider making your fetch work with a button
+const characters = document.querySelectorAll(".character img")
+const button = document.querySelector('#random')
+button.addEventListener('click', () =>{
+    axios({
+        method: 'get',
+        url: 'https://rickandmortyapi.com/api/character/1'
```

Comment:
You can an array with the URL and git the result for all five characters,
that is better then send an indvasul request each time.